redbox-core/redbox/chains/ingest.py

```diff
@@ -43,5 +43,5 @@ def ingest_from_loader(
         document_loader(document_loader_type=document_loader_type, s3_client=s3_client, env=env)
         | RunnableLambda(list)
         | log_chunks
-        | RunnableLambda(partial(vectorstore.aadd_documents, create_index_if_not_exists=False))  # type: ignore[arg-type]
```

Comment:
can we leave this as the async version? I don't think it matters hugely but if we can async we should